packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -897,6 +897,20 @@ class _DraggableScrollableSheetScrollPosition extends ScrollPositionWithSingleCo
     }
   }
 
+  @override
+  void pointerScroll(double delta) {
+    print("call to pointerScroll");
```

Comment:
I tried this with
- mouse with physical scroll wheel
  - calls pointerScroll as expected
  - sheet does not expand or collapse
- Apple magic mouse (trackpad)
  - does not call pointerScroll, also expected
  - sheet expands and collapses as expected
- Stand alone trackpad
  - does not call pointerScroll, also expected
  - sheet expands and collapses as expected

Reply:
Let's chat about this more later today and we can see about reproducing the behavior you described.